README.md

```diff
@@ -13,35 +13,64 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
 ## Prebuilt Container
 An easy way to get started with ONNX-MLIR is to use a prebuilt docker image. These images are created as a result of a successful merge build on the trunk.
 This means that the latest image represents the tip of the trunk.
-Currently there are images for amd64, ppc64le and IBM System Z respectively saved in Docker Hub as onnxmlirczar/onnx-mlir-build:amd64,
-onnxmlirczar/onnx-mlir-build:ppc64le and onnxmlirczar/onnx-mlir-build:s390x. To use one of these images either pull it directly from Docker Hub,
-launch a container and run an interactive bash shell in it, or use it as the base image in a dockerfile. The container contains the full build tree including
-the prerequisites and a clone of the source code. The source can be modified and onnx-mlir rebuilt from within the container, so it is possible to use it
-as a development environment. It is also possible to attach vscode to the running container. An example Dockerfile and vscode configuration files can be
-seen in the docs folder. The Dockerfile is shown here.
+Currently there are both Release and Debug mode images for amd64, ppc64le and IBM System Z saved in Docker Hub as, respectively, onnxmlirczar/onnx-mlir
+and onnxmlirczar/onnx-mlir-dev. To use one of these images either pull it directly from Docker Hub,
+launch a container and run an interactive bash shell in it, or use it as the base image in a dockerfile. The onnx-mlir container just contains the 
+built compiler and can be used to compile models. The onnx-mlir-dev container contains the full build tree including the prerequisites and a clone
```

Comment:
`The onnx-mlir container just contains ...` should be `The onnx-mlir image just contains ...`
`The onnx-mlir-dev container contains ...` should be `The onnx-mlir-dev image contains ...`

Reply:
yes indeed.